src/components/back-link/index.njk

```diff
@@ -0,0 +1,49 @@
+---
+title: Back link
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+
+The back link reuses the same interface and parameters as the [Breadcrumbs](/components/breadcrumb) component.
+Always place back links before a page’s `<main>` content element.  This makes sure the [Skip to content](/components/skip-to-content) link allows the user to skip all navigation, including the back link.
+Where possible this should be set in the [page template](/styles/page-template/) in `pageConfig`. This will ensure it will have the correct placing on the page.
```

Comment:
```suggestion
The back link reuses the same interface and parameters as the [Breadcrumbs](/components/breadcrumb) component.

Always place back links before a page’s `<main>` content element.  This makes sure the [Skip to content](/components/skip-to-content) link allows the user to skip all navigation, including the back link.
Where possible this should be set in the [page template](/styles/page-template/) in `pageConfig`. This will ensure it will have the correct placing on the page.
```

Reply:
Start the documentation with: 'A back link is used to help the user go back to the previous page of a transaction or questionnaire.'